docs/reference/technology/USB.md

```diff
@@ -56,11 +56,11 @@ USB defines 5 separate states a device can be in - Attached, Powered, Default, A
 | Address    |    No new functionality                  |
 | Configured |    All enabled endpoints are functional  |
 
-At any time, the USB device can enter a state with less functionality. This could be due to a loss of power event or a surprise USB disconnect. When leaving or outside of the Configured state, USBDevice ignores writes to and reads from all endpoints other than endpoint 0.
-
+The USB device can enter a state with less functionality at any time. This could be due to a loss of power or surprise disconnect. When leaving or outside of the Configured state, USBDevice ignores writes to and reads from all endpoints other than endpoint 0.
+<!---initially confusing wording--->
 ## USB component callbacks
 
-All callbacks USBDevice sends to its children are prefixed with callback_*. USBDevice calls these callbacks with the USB lock held. One notable callback is `callback_state_change`, which USB components can use generically to handle leaving the Configured state. The USB stack automatically exits the Configured state on disconnect, power loss or USB reset.
+All callbacks that USBDevice sends to its children are prefixed with `callback_*`. USBDevice calls these callbacks with the USB lock held. One notable callback is `callback_state_change`, which USB components can use generically to handle leaving the Configured state. The USB stack automatically exits the Configured state on disconnect, power loss or USB reset.
```

Comment:
This "that" is not necessary. Good catch on the tick marks.

Reply:
```suggestion
All callbacks USBDevice sends to its children are prefixed with `callback_*`. USBDevice calls these callbacks with the USB lock held. One notable callback is `callback_state_change`, which USB components can use generically to handle leaving the Configured state. The USB stack automatically exits the Configured state on disconnect, power loss or USB reset.
```